c++/src/kj/compat/http.c++

```diff
@@ -2594,12 +2663,167 @@ private:
 
     static constexpr byte FIN_MASK = 0x80;
     static constexpr byte RSV_MASK = 0x70;
+    static constexpr byte RSV1_MASK = 0x40;
     static constexpr byte OPCODE_MASK = 0x0f;
 
     static constexpr byte USE_MASK_MASK = 0x80;
     static constexpr byte PAYLOAD_LEN_MASK = 0x7f;
   };
 
+  class Context {
```

Comment:
Could this class be simplified by using `kj::GzipOutputStream` on top of `kj::VectorOutputStream` or the like? Especially perhaps if #1428 got finished and merged, which avoids the need for `GzipOutputStream` to hold its own buffer (which is something you're currently doing anyway in this class, but which I commented we might want to avoid...).

Reply:
@kentonv Hi, I'm the author of #1428 and as far as I'm concerned it's finished unless you have further comments on it which I'll be happy to look at. We've been using that code in our branch of Cap'n Proto for months now and it's working fine.